configs/top_down/resnet/crowdpose/res101_crowdpose_256x192.py

```diff
@@ -123,13 +123,13 @@
     workers_per_gpu=2,
     train=dict(
         type='TopDownCrowdPoseDataset',
-        ann_file=f'{data_root}/annotations/crowdpose_train.json',
+        ann_file=f'{data_root}/annotations/crowdpose_trainval.json',
         img_prefix=f'{data_root}/images/',
         data_cfg=data_cfg,
         pipeline=train_pipeline),
     val=dict(
         type='TopDownCrowdPoseDataset',
-        ann_file=f'{data_root}/annotations/crowdpose_val.json',
+        ann_file=f'{data_root}/annotations/crowdpose_test.json',
```

Comment:
is this a common practice in crowdpose, that using test as val?

Reply:
Already fixed it.